synapse/handlers/oidc.py

```diff
@@ -935,17 +954,38 @@ async def handle_redirect_request(
 
         state = generate_token()
         nonce = generate_token()
+        code_verifier = ""
```

Comment:
I wondered why you weren't making that optional, but then I remembered we rely on macaroons for those... :'(

Reply:
Yeahhhh, I figured this was the cleanest way to do it.